docs/modules/ROOT/pages/explanations/decisions/mariadb.adoc

```diff
@@ -19,21 +19,24 @@ As well as the Bitnami Helm charts.
 
 The following section contains the operators that have been looked at.
 
-[cols="1,1,1"]
+[cols="1,1,1,1"]
 |===
-|Requirements |https://github.com/abalki001/mariadb-operator[Official MariaDB Operator] |https://github.com/bitnami/charts/tree/master/bitnami/mariadb-galera[Bitnami Helmchart]
+|Requirements |https://github.com/abalki001/mariadb-operator[Official MariaDB Operator] |https://github.com/bitnami/charts/tree/master/bitnami/mariadb-galera[Bitnami Helmchart]| https://github.com/mariadb-operator/mariadb-operator[New Mariadb Operator]
```

Comment:
* Official MariaDB Operator: Why is this called "official"? I can't find anything about that one being "official". Please find a better name, e.g. "Persistentsys MariaDB Operator" (found in the API group name)
* New Mariadb Operator: Can we just call this "mariadb-operator" like it is named in the project?